config/workflows/templates/15/job.xml

```diff
@@ -0,0 +1,331 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.7"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.7 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.7/schedulerjob.xsd"
+    name="Twitter Language R Analysis" projectName="3. Big Data Workflows"
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="1"
>
+  <variables>
+    <variable name="KeyWord" value="HybridCloud"/>
+    <variable name="Number_of_search" value="3"/>
+    <variable name="Number_of_tweets" value="2000"/>
+  </variables>
+  <description>
+    <![CDATA[ A workflow to run a simple Twitter analysis using R ]]>
+  </description>
+  <taskFlow>
+    <task name="Analysis">
+      <description>
+        <![CDATA[ Analysis' body. ]]>
+      </description>
+      <depends>
+        <task ref="Split"/>
+      </depends>
+      <forkEnvironment javaHome="/usr" >
+        <envScript>
+          <script>
+            <code language="python">
+              <![CDATA[
+#Be aware, that the prefix command is internally split by spaces. So paths with spaces won't work.
+# Prepare Docker parameters 
+containerName = 'thomasmoulin/demor' 
+dockerRunCommand =  'docker run ' 
+dockerParameters = '--rm ' 
+# Prepare ProActive home volume 
+paHomeHost = variables.get("PA_SCHEDULER_HOME") 
+paHomeContainer = variables.get("PA_SCHEDULER_HOME") 
+proActiveHomeVolume = '-v '+paHomeHost +':'+paHomeContainer+' ' 
+# Prepare working directory (For Dataspaces and serialized task file) 
+workspaceHost = localspace 
+workspaceContainer = localspace 
+workspaceVolume = '-v '+localspace +':'+localspace+' ' 
+# Prepare container working directory 
+containerWorkingDirectory = '-w '+workspaceContainer+' ' 
+# Save pre execution command into magic variable 'preJavaHomeCmd', which is picked up by the node 
+preJavaHomeCmd = dockerRunCommand + dockerParameters + proActiveHomeVolume + workspaceVolume + containerWorkingDirectory + containerName
+]]>
+            </code>
+          </script>
+        </envScript>
+      </forkEnvironment>
+      <pre>
+        <script>
+          <code language="bash">
+            <![CDATA[
+
+]]>
+          </code>
+        </script>
+      </pre>
+      <scriptExecutable>
+        <script>
+          <code language="Language R">
+            <![CDATA[
+#Change to your own or the limits will be shared
+consumer_key_api <- "K5sM7uI8QPwK1iiTsvWayIl9q"
+consumer_secret_api <- "4Lc9V9bjzNTI2B6Eq7wvf3TZJQMnv3sFqiDCBCgqWQL1kjFmM6"
+token <- "804452393895088129-JmyRNVoDLaDBOyjlUrh4my3Yl8M2Q9U"
+token_secret <- "yedHoQyQEardwoQNbFraWIz1n552BgfADDk5XOSwcc7PT"
+#15 calls every 15 minutes. 
```

Comment:
An account created just for this. Since no-one cares about it, security is a non-issue.
It was either that or having an unusable template.